README.md

```diff
@@ -22,9 +22,11 @@ The connector is great for getting started with Conduit but also for certain typ
 
 ### Configuration
 
-| name          | description                                                                                | required | Default |
-|---------------|--------------------------------------------------------------------------------------------|----------|---------|
-| recordCount   | Number of records to be generated. -1 for no limit.                                        | false    | "-1"    |
-| readTime      | The time it takes to 'read' a record.                                                      | false    | "0s"    |
-| fields        | A comma-separated list of name:type tokens, where type can be: int, string, time, bool.    | true     | ""      |
-| format        | Format of the generated payload data: raw, structured.                                     | false    | "raw"   |
+| name         | description                                                                                | required | Default             |
+|--------------|--------------------------------------------------------------------------------------------|----------|---------------------|
+| recordCount  | Number of records to be generated. -1 for no limit.                                        | false    | "-1"                |
+| readTime     | The time it takes to 'read' a record.                                                      | false    | "0s"                |
+| sleepTime    | The time the generator 'sleeps' before it starts generating records. Must be non-negative. | false    | "0s"                |
+| generateTime | The amount of time the generator is generating records. Must be positive.                  | false    | max. duration in Go |
```

Comment:
`generateTime` won't make a difference unless `sleepTime` is specified as well. This should also be noted.

Reply:
Good one, just pushed the changes.